invenio_files_rest/views.py

```diff
@@ -267,6 +267,16 @@ def inner(self, bucket, key, upload_id, *args, **kwargs):
     return decorate
 
 
+def ensure_input_stream_is_not_read(f):
```

Comment:
Minor and IMHO: `ensure_input_stream_is_not_read` -> `ensure_input_stream_is_not_exhausted`?

Same for the test if changed.